core/src/test/java/org/testcontainers/junit/wait/strategy/AbstractWaitStrategyTest.java

```diff
@@ -0,0 +1,93 @@
+package org.testcontainers.junit.wait.strategy;
```

Comment:
Hm... I'm not sure this changes are related to the PR

Reply:
It's somewhat needed to test the new methods I added. See `strategy/HttpWaitStrategyTest.java` class.

Reply:
I think we should just start using new implementation (under `org.testcontainers.containers.wait.strategy` package) in `HttpWaitStrategyTest` & `AbstractWaitStrategyTest` since the old classes are just wrappers now

Reply:
@dadoonet ping :)

Reply:
Hum. I'm missing something. I probably misunderstood what you are expecting.

Reply:
You just duplicated an existing test, right?

Reply:
True.

Reply:
But I added a call to my method

Reply:
But I added a call to my method

Reply:
We should just fix the imports in the old test and start using new wait strategies, so that you can also add your new method to that test